src/webgpu/shader/execution/expression/call/builtin/textureStore.spec.ts

```diff
@@ -38,6 +38,7 @@ Parameters:
   )
   .params(u =>
     u
+      .beginSubcases()
       .combine('F', TexelFormats)
```

Comment:
You may want to have texel format be a non-subcase parameter, since it would probably be relatively common for an implementation to be missing support for a texel format (e.g. if we later add a new format that can be used for storage and the test gets implemented before all the browsers implement it). We do this for texture formats in most tests for this reason.

---

As a side note, the "shader representation" part of TexelFormats isn't a necessary part of the identifying info of the testcase. So in this case we would usually either write something like:

- use an underscore to make it not part of the "public"/"identifying" parameters
```
export const kTexelFormatParams = [
  { format: 'rgba8unorm', _shaderType: 'f32' },
  // ...
];
```

`.params(u => u.combineWithParams(kTexelFormatParams)`

- look up the shader type separately (in roughly the style of `capability_info.ts`)
```
export const kStorageTextureFormatInfo = {
  'rgba8unorm': { shaderType: 'f32' },
  // ...
];
export const kStorageTextureFormats = keysOf(kStorageTextureFormatInfo);
```

`.params(u => u.combine('F', kStorageTextureFormats)`

`const { shaderType } = kStorageTextureFormatInfo[t.params.F];`

Reply:
Perfect, thanks. Done.